apps/rush-lib/src/logic/pnpm/PnpmProjectDependencyManifest.ts

```diff
@@ -211,4 +236,27 @@ export class PnpmProjectDependencyManifest {
       }
     }
   }
+
+  private _parsePeerDependencyKeysFromSpecifier(specifier: string): { [peerDependencyName: string]: string } {
```

Comment:
Can we describe the pattern on the param specifier here? Maybe include a few examples?